gradle/libs.versions.toml

```diff
@@ -113,6 +113,8 @@ androidx-appcompat = { module = "androidx.appcompat:appcompat", version.ref = "a
 
 androidx-cardview = { module = "androidx.cardview:cardview", version.ref = "androidx-cardview" }
 
+androidx-compose-compiler = { module = "androidx.compose.compiler:compiler", version.ref = "androidx-compose-compiler" }
```

Comment:
We use the version reference in the Android convention plugin, but that isn't enough for Renovate to pick it up.  So by defining the actual maven artifact here, Renovate will see it and include it in its update checks.